### PR TITLE
Resolves #1528. | Add /ai command back.

### LINKED ISSUE #1528: ai not working
please fix /ai because it doesn't work

### PR DESCRIPTION
**Pull request by xItsNickHD**
You can edit what I made in /ai, I don't really mind, AI is actually pretty useful to me.

Edit: 075e62d
#1528

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_admininfo.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import net.md_5.bungee.api.ChatColor;
+
+@CommandPermissions(level = AdminLevel.OP, source = SourceType.ONLY_IN_GAME, blockHostConsole = true)
+@CommandParameters(
+		description = "Learn how to apply for admin.", 
+		usage = "/<command> [on | off]",
```
Comment (AvalancheYT) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_admininfo.java]: Ahh, why is there on and off in usage.

Comment (xItsNickHD) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_admininfo.java]: Apologies, I was going to remove that until I forgot to.

**Comment by xItsNickHD** [comment]
075e62d

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_admininfo.java (hunk 1 above)
Comment (xItsNickHD) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_admininfo.java]: Yes, I patched that; 075e62d

**Comment by AvalancheYT** [comment]
FYI, this PR have not met the requirement. You must have an associated issue (Accepted) for each PR.

**Comment by Wild1145** [comment]
Even if this did have an associated issue these sorts of commands are not accepted, its just a wall of text, TFM does not want that.

**Comment by DUFCLiam** [comment]
Denied due to the new plugin made by @Wild1145 requesting closure of this PR @JeromSar

**Comment by Adaptified** [comment]
Please stop acting like a developer of the repo, @DUFCLiam.

**Comment by JeromSar** [comment]
@DUFCLiam With all due respect, you have no authority here.

**Comment by JeromSar** [comment]
This pull request does not meet the [pull request requirements](https://github.com/TotalFreedom/TotalFreedomMod/wiki/Contributing#pull-request-requirements) and therefore is closed.

**Pull request closed without merging**